atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schemas.java

```diff
@@ -83,6 +83,12 @@ public static boolean isTableNameValid(String tableName) {
         return true;
     }
 
+    public static int getInternalTableNameLength(String tableName, Namespace namespace) {
+        // The table name computed at run-time is (generally) the concatenation of the namespace name
```

Comment:
The manner in which the table name is computed is different for Postgres, mostly due to a table name prefix. Would be happy to discuss more in-depth in person. Maybe, the method should live in `Schema` as it is used only there.